module/VuFind/src/VuFind/Db/Service/RatingsService.php

```diff
@@ -126,4 +126,45 @@ public function addOrUpdateRating(
             ? $this->getDbTable('resource')->select(['id' => $resourceOrId])->current() : $resourceOrId;
         return $resource->addOrUpdateRating(is_int($userOrId) ? $userOrId : $userOrId->getId(), $rating);
     }
+
+    /**
+     * Delete ratings by given user and rating ids
```

Comment:
Add punctuation for consistency.

```suggestion
     * Delete ratings by given user and rating ids.
```